quantum/os_detection.c

```diff
@@ -133,7 +133,7 @@ void process_wlength(const uint16_t w_length) {
         } else if (setups_data.count == setups_data.cnt_ff) {
             // Linux has 3 packets with 0xFF.
             guessed = OS_LINUX;
-        } else if (setups_data.count == 5 && setups_data.last_wlength == 0xFF && setups_data.cnt_ff == 1 && setups_data.cnt_02 == 2) {
+        } else if (setups_data.count >= 5 && setups_data.last_wlength == 0xFF && setups_data.cnt_ff >= 1 && setups_data.cnt_02 >= 2) {
```

Comment:
> // Some collected sequences of wLength can be found in tests.

Any chance we can collect a sample of the "new" macos behaviour and update tests?

Reply:
added to comments and tests